cni/pkg/install-cni/pkg/install/binaries.go

```diff
@@ -0,0 +1,77 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/etcd/pkg/fileutil"
+
+	"istio.io/istio/cni/pkg/install-cni/pkg/constants"
+	"istio.io/istio/cni/pkg/install-cni/pkg/util"
+	"istio.io/pkg/log"
+)
+
+func copyBinaries(updateBinaries bool, skipBinaries []string) error {
+	srcDir := constants.CNIBinDir
+	targetDirs := []string{constants.HostCNIBinDir, constants.SecondaryBinDir}
+
+	skipBinariesSet := arrToSet(skipBinaries)
+
+	for _, targetDir := range targetDirs {
+		if fileutil.IsDirWriteable(targetDir) != nil {
+			log.Infof("Directory %s is not writable, skipping.", targetDir)
+			continue
+		}
+
+		files, err := ioutil.ReadDir(srcDir)
+		if err != nil {
+			return err
+		}
+
+		for _, file := range files {
+			filename := file.Name()
+			if skipBinariesSet[filename] {
+				log.Infof("%s is in SKIP_CNI_BINARIES, skipping.", filename)
```

Comment:
Sounds it is a DEBUG log. Why `SKIP_CNI_BINARIES` uppercase?

Reply:
These are environment variables, and this output was in the shell script so I kept it in the translation. I could remove these logs though? @rlenglet @jwendell

Reply:
No, please. Let's keep the compatibility with the shell script. If we remove this we might break some installation script out there.